kahuna/public/js/search/results.js

```diff
@@ -154,6 +154,9 @@ results.controller('SearchResultsCtrl', [
             }
 
             return images;
+        }).catch(error => {
+            ctrl.loadingError = error;
```

Comment:
Just an observation, nothing of particular interest... the [global banner message](https://github.com/guardian/grid/blob/master/kahuna/public/js/errors/http.js) should also show in these cases too.